src/app/test/factories/test-entity.factory.ts

```diff
@@ -2,6 +2,7 @@ import { Part } from '../../core/models/part'
 import { Segment } from '../../core/models/segment'
 import { Rundown } from '../../core/models/rundown'
 import { Piece } from '../../core/models/piece'
+import { RundownTimingType } from '../../core/enums/rundown-timing-type'
 
 export class TestEntityFactory {
   public createRundown(rundown: Partial<Rundown> = {}): Rundown {
```

Comment:
All the methods in here is only used for creating test entities in our tests.
This means that the public methods can be static. Then you don't have to instantiate a new `TesteEntityFactory` in every single test. (We do the same in the backend ;) )